app/models/contribution.rb

```diff
@@ -170,4 +172,42 @@ def set_oai_pmh_fields
   def to_param
     ore_aggregation.edm_aggregatedCHO.uuid
   end
+
+  def to_rdf
+    serialised_rdfxml_graph || ore_aggregation.to_rdf
```

Comment:
how about having this create a serialization when the graph was generated?

Reply:
That would be duplicating the responsibility for doing so as there should already be a job queued. This fallback is just in case a request for the RDF comes in before the job is processed.